src/components/ui/AvatarInput/AvatarInput.tsx

```diff
@@ -22,7 +25,9 @@ const AvatarInput = ({
   title,
   imagePath,
   setImagePath,
+  imageContainerStyles = '',
   buttonTitle = 'Upload',
+  buttonStyles = '',
```

Comment:
you can exclude = '', clsx resolve cases where there is no styles in props

Reply:
done